src/sgame/sg_struct.h

```diff
@@ -624,6 +624,7 @@ struct level_locals_s
 	bool restarted; // waiting for a map_restart to fire
 
 	int      numConnectedClients; // connected
```

Comment:
while you're at it, maybe you can also edit that other comment as `// connected clients (players + bots)` or something like that, to be more explicit.

Reply:
Sure, I can do that too :)